apps/pwabuilder/src/script/pages/app-report.ts

```diff
@@ -1654,7 +1654,7 @@ export class AppReport extends LitElement {
     navigator.clipboard.writeText(window.location.href).then(() => {
```

Comment:
async await would make this easier to read imo, but not a biggie

Reply:
wasn't actually meant to be in this PR